staging/cert-manager-setup/files/crds.yaml

```diff
@@ -7512,4 +7511,3 @@ status:
   conditions: []
```

Comment:
Can we remove `"helm.sh/hook": crd-install` annotation?

Reply:
oh, i hit approval accidentally

Reply:
No the crd-install hook still needs to be used here.

Reply:
yeah, still needs to be here.

Reply:
Hum? Why?

Reply:
i believe helm 2 uses it to check whether it's a CRD or not for validations.

Reply:
oh, right. OK.